src/Database/Driver/OracleBase.php

```diff
@@ -388,14 +390,14 @@ protected function _autoincrementSequenceId(?string $table, ?string $column)
             $tableName = strtoupper($table);
             $columnName = strtoupper($column);
         }
-        $query = "select sequence_name from user_tab_identity_cols where table_name='$tableName' and column_name='$columnName'";
+        $query = "select sequence_name from all_tab_identity_cols where table_name='$tableName' and column_name='$columnName'";
         $this->connect();
         $seqStatement = $this->_connection->query($query);
         $result = $seqStatement->fetch(PDO::FETCH_NUM);
 
         $sequenceName = $result[0];
 
-        $statement = $this->_connection->query("SELECT {$sequenceName}.CURRVAL FROM DUAL");
+        $statement = $this->_connection->query("SELECT {$schema}.{$sequenceName}.CURRVAL FROM DUAL");
```

Comment:
If `$schema` is an optional parameter, we cannot presume it to be set here.

Reply:
This is fixed by checking if the schema is set. 
If it is then the new statement is called, if its not then the original is called.